src/FormMigration/Steps/PaymentGateways.php

```diff
@@ -10,6 +10,7 @@
 class PaymentGateways extends FormMigrationStep
 {
     /**
+     * @unreleased Add Per Form Gateways settings to this step
```

Comment:
I would suggest adding a new migration step for the Per Form Gateway settings.

Reply:
Resolved in c99e07a84ef766f41f831adf7b930b60e75e893b